src/server.tsx

```diff
@@ -27,16 +27,28 @@ const DEV = !PROD;
 const RESTRICTIVE_ROBOTS_TXT = ['User-agent: *',
   'Disallow: /', ''].join('\n');
 
-const OPEN_ROBOTS_TXT = getOpenRobotsTxt();
-
 const template = fs.readFileSync('./dist/layout.html', 'utf8');
 
-function renderHtml(layout: string, title: string, description: string, reactHtml: string) {
+function renderHtml(layout: string, metadata: IPageMetadata, reactHtml: string) {
   let result = layout;
-  result = result.replace('<title>FreeBoardGame.org</title>', `<title>${title}</title>`);
-  result = result.replace(
-    '<meta name="Description" content="">',
-    `<meta name="Description" content="${description}">`);
+
+  result = result.replace('<title>FreeBoardGame.org</title>', `<title>${metadata.title}</title>`);
+
+  if (metadata.description) {
+    result = result.replace(
+      '    <meta name="description" content="">\n',
```

Comment:
can we remove the spaces inside the string being replaced? I think this is a very easy source of failure.

Reply:
Removed, it was so the HTML was aligned